trappy/plotter/ILinePlotGen.py

```diff
@@ -34,6 +34,18 @@
 
 from IPython.display import display, HTML
 
+def df_to_dygraph(data_frame):
```

Comment:
`data_dict` is not always a dataframe, it can be a dictionary.  Try to plot only one column.  In that case, you can't do `data_frame.values` or `data_frame.index`.

The problem is in `ILinePlot::_fix_indexes()`, it can return a data_frame and it shouldn't.  I think that before this patch we should get `_fix_indexes()` to return a dict and make this function deal with dicts instead of dataframes.